### PR TITLE
Ztisnes/0.1.0

### PR DESCRIPTION
**Pull request by 0auBSQ**
Retriving database setup files before working IITQ-15

### CODE REVIEW

**Review by jeremie-gauthier — CHANGES_REQUESTED** [review]
We should remove express from this project to get the best from NextJS build time optimizations.
Here is an useful link for implementing some auth strategies with Next in a serverless style: 
https://next-auth.js.org/getting-started/introduction

File: package.json [hunk 1]
```diff
@@ -3,7 +3,7 @@
   "version": "0.1.0",
   "private": true,
   "scripts": {
-    "dev": "next dev",
+    "dev": "npm i next && next dev",
```
Comment (jeremie-gauthier) [inline package.json]: revert this change pls, install and run are 2 differents process.
this command will slow down the start up time

To remove express, run `npm remove express`

**Review by 0auBSQ — COMMENTED** [review]
Removed a deprecated file that was only placeholder (using express)

**Review by jeremie-gauthier — CHANGES_REQUESTED** [review]

**Comment by jeremie-gauthier** [comment]
@all-contributors please add @0auBSQ for code
@all-contributors please add @zedin27 for code

**Comment by jeremie-gauthier** [comment]
@all-contributors please add @0auBSQ for code

**Comment by jeremie-gauthier** [comment]
@all-contributors please add @zedin27  for code

**Review by jeremie-gauthier — APPROVED** [review]

**Pull request merged**